LfuAlgoritm/src/main/java/com/task2/service/LfuAlgoritm.java

```diff
@@ -0,0 +1,66 @@
+package main.java.com.task2.service;
+
+import main.java.com.task2.dao.Database;
+import main.java.com.task2.model.Element;
+import main.java.com.task2.model.Menu;
+import main.java.com.task2.model.News;
+
+import java.util.*;
+
+public class LfuAlgoritm {
```

Comment:
I think this is a bad name for a class.

Reply:
I changed class name to "LfuImplementation".